sections/hoc/PageLayout.jsx

```diff
@@ -22,8 +23,27 @@ export const PageLayout = ({ children, ...props }) => {
 };
 
 const Wrapper = styled.div`
+  background-image: url("/home/background-large.svg");
```

Comment:
This is too specific use case to just the Top Section. There are a lot of different blobs and you should think of how to make this reusable (props) with other sections.

I also think this is not a great component to put it in. It's not really a PageLayout but a ComponentLayout. This should be a new component to wrap around each component (not each page) and you should consider the use case that the component does not have a background? If you look carefully at what PageLayout does is it wraps the children with a navbar and footer.

Reply:
Definitely agree with Sharon, this component should be its own section wrapper (each section a page can specify which blob background svg to use).

It looks great for the top section of the home page, but even on the home page, there are other sections that require the background svg component. Every page uses the PageLayout component to include the Navbar and Footer, but not every page (or even every section) should have the same background svg as the top section of the first page 
![image](https://user-images.githubusercontent.com/65516245/128110743-ed5aabc4-9d7f-41d1-b940-885a071b6061.png)

what /resources looks like with background component in PageLayout: 
![image](https://user-images.githubusercontent.com/65516245/128110855-294cc626-c8d5-4d6f-8aef-b71f16b84ec1.png)

what /resources should look like (using different blob) 
![image](https://user-images.githubusercontent.com/65516245/128110816-76c427c9-f5a2-42ce-8ad1-881295f2eac7.png)